CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to seek-style-guide
+
+⚠️ 🌏 👀 First and foremost, remember that this repo is **open source**.
+
+Don't post anything or commit any code that isn't ready for the entire world to see. If you're introducing new patterns, try to keep it abstract, rather than making specific reference to the features under development. While a lot of this information is probably harmless, it's better to be safe than sorry.
+
+If you work at SEEK and run into issues along the way, or even if you find some of these steps confusing or intimidating, please reach out to your friendly neighbourhood developer in the style guide development Slack channel. We'll be super excited to help out!
+
+## Setup
+
+First, install [Node.js](https://nodejs.org/).
+
+After cloning the project, install the dependencies:
+
+```bash
+$ npm install
+```
+
+## Workflow
+
+Before starting your work, first ensure you're in the `master` branch and that you've pulled down the latest changes:
+
+```bash
+$ git checkout master
+$ git pull
+```
+
+Next, create a new branch for your work, with an appropriate name for your change:
+
+```bash
+$ git checkout -b add-my-cool-new-feature
+```
+
+In order to see changes to the style guide during development, run the local development server:
+
+```bash
+$ npm start
```

Comment:
We don't even have a start for this repo, not sure we will either. Maybe remove?